### PR TITLE
py3(django): defer more model imports until after Django setup (part 2, take 3)

### PR DESCRIPTION
**Pull request by joshuarli**
Retry of https://github.com/getsentry/sentry/pull/15094 except that I missed this one import and it happened to not be covered by tests that are outside of the py3 upgrade scope to fix.

Will need to go through canary just to make sure things are frosty.

### CODE REVIEW

**Review by wedamija — APPROVED** [review]

**Review by tkaemming — APPROVED** [review]
Didn't look, just unblocking as code owner since there there no way to dismiss the review requirement and @wedamija already approved.

**Pull request merged**
